bedrock/base/templates/404.html

```diff
@@ -2,47 +2,63 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% set_lang_files "mozorg/404" %}
-
-{% extends "base-pebbles.html" %}
+{% set_lang_files "mozorg/404-update-2020" %}
+{% extends "base-error.html" %}
 
 {% block gtm_page_id %}data-gtm-page-id="404"{% endblock %}
 
 {% block page_title %}{{ _('404: Page Not Found') }}{% endblock %}
 
-{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
-{% block canonical_urls %}{% endblock %}
-
 {% block page_css %}
   {{ css_bundle('page_not_found') }}
 {% endblock %}
 
+{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
+{% block canonical_urls %}{% endblock %}
+
 {% block content %}
-<main>
-  <div class="content">
-    <header>
-      <h1><span>{{ _('Whoops!') }}</span></h1>
-      <p>{{ _('Did you make a left at that last URL instead of a right? No problem. Here are some tips to get you back on your way:') }}</p>
-    </header>
-    <div class="error-content">
-      <ul class="prose">
-        <li>{{ _('If you typed in the address, check your spelling. Could just be a typo.') }}</li>
+  <main role="main">
+    <section class="mzp-l-content message-section">
+      <h2>{{ _("Sorry, we can't find that page") }}</h2>
```

Comment:
Nit: single quotes for wrapping L10n strings, and use curly apostrophes inside copy.
```suggestion
      <h2>{{ _('Sorry, we can’t find that page') }}</h2>
```

Reply:
Should this also be an `h1`?